packages/back-end/src/controllers/auth.ts

```diff
@@ -276,7 +276,9 @@ export async function postFirstTimeRegister(
     });
   }
 
-  const user = await createUser(name, email, password);
+  // grant the first user on a new installation super admin access
+  const user = await createUser({ name, email, password, superAdmin: true });
```

Comment:
Digging into this endpoint more, `isNewInstallation` uses an in-memory cache, so it's possible in a horizontally scaled environment it could return `true` when there are actually already orgs/users.

Looking through the code, I don't think that in-memory cache is valuable.  `isNewInstallation` is not called that often and it's only doing 2 efficient mongo lookups.

So, I'd recommend removing the caching layer to reduce the chance of bugs here and elsewhere.

Reply:
Removed the cache layer from `isNewInstallation` and the use of `markInstalled` entirely since the former method will check every time.